cognite_toolkit/_cdf_tk/load/_resource_loaders.py

```diff
@@ -1899,3 +1909,114 @@ def count(self, ids: SequenceNotStr[NodeId]) -> int:
     def drop_data(self, ids: SequenceNotStr[NodeId]) -> int:
         # Nodes will be deleted in .delete call.
         return 0
+
+
+@final
+class WorkflowLoader(ResourceLoader[str, WorkflowUpsert, Workflow, WorkflowUpsertList, WorkflowList]):
+    api_name = "workflows"
+    folder_name = "workflows"
+    filename_pattern = r"^(?!.*\b(version|WorkflowVersion)\b).*"
```

Comment:
This is surprising to me. I expected based on our previous discussion that this should check for files that ends with `.Workflow`? and not files that does not end with `WorkflowVersion` or `version`. Any reason why this changed?

```suggestion
    filename_pattern = r"\.Workflow$"
```

Reply:
No good reason, just remains of old code.